server/routes/users.js

```diff
@@ -2,12 +2,17 @@ import { Router } from 'express';
 import User from '../controllers/user';
 import inputValidator from '../middlewares/validations';
 import checkAuth from '../middlewares/checkAuth';
```

Comment:
checkAuth should be `isAuthenticated`. Please change it

Reply:
okay. Thanks